test/test.tf

```diff
@@ -26,3 +26,35 @@ module "aws_deploy-test" {
   enable_state_channels = true
   enable_internal_api   = true
 }
+
+module "aws_deploy-test_vpc" {
+  source            = "../"
+  env               = var.env_name
+  envid             = "${var.envid}"
```

Comment:
Can you fix all "simple assignments" to not use "${}", as currently it is a mix of both. At least on the parts where you already did changes anyway

Reply:
82cada4